components/Extensions/samples/ListViewExtensions.md

```diff
@@ -124,7 +124,7 @@ await MyGridView.SmoothScrollIntoViewWithItemAsync(item: object, itemPlacement:
 ### Methods
 
 | Methods | Return Type | Description |
-| -- | -- | -- |
+|---------|-------------|-------------|
```

Comment:
This is an old-style page, we should just re-write.

Reply:
Have spruced up this page in an upcoming commit to #214. Removed these and added the SmoothScroll as an actual sample.